src/main/java/org/balanpy/HigieneController.java

```diff
@@ -1,12 +1,219 @@
+
+	@FXML
+	private TextField ultimoBano;
+
+	@FXML
+	private TextField ultimoCepillado;
+
+	@FXML
+	private TextArea productosNR;
+
+	@FXML
+	private TextArea ultimaDesp;
+
+	public HigieneController() {
+
+		try {
+			ObjectMapper om = new ObjectMapper();
+
+			mascotas = om.readValue(MASCOTAS_PATH.toFile(), new TypeReference<ArrayList<MascotaImpl>>() {
+
+			});
+
+		} catch (IOException e) {
+
+			System.out.println("Failed to load pet data. " + e.getMessage());
+
+			mascotas = new ArrayList<MascotaImpl>();
+
+		}
+
+	}
+
+	public void selectProfile(int id) throws Exception {
+
+		ultimoBano.setText(ultimoBanoToString(mascotas.get(id)));
+
+		ultimoCepillado.setText(ultimoCepilladoToString(mascotas.get(id)));
+
+		ultimaDesp.setText(desparasitadoToString(mascotas.get(id)));
+
+		productosNR.setText(productosToString(mascotas.get(id)));
+
+	}
+
+	public void selectProfile0(MouseEvent event) throws Exception {
+
+		selectProfile(0);
+	}
+
+	public void selectProfile1(MouseEvent event) throws Exception {
+
+		selectProfile(1);
+
+	}
+
+	public void selectProfile2(MouseEvent event) throws Exception {
+
+		selectProfile(2);
+
+	}
+
+	private void setProfile(ImageView imageView, Label label, Mascota mascota) throws MalformedURLException {
+
+		Path imagePath = Paths.get(Configuracion.getRootDir(), mascota.getUUID().toString(), "profile.png");
+
+		label.setText(mascota.getNombre());
+
+		imageView.setImage(new Image(imagePath.toUri().toURL().toExternalForm()));
+
+	}
+
+	private String ultimoBanoToString(Mascota mascota) {
+
+		String bano = mascota.getHigiene().getBano();
+
+		return bano;
+
+	}
+
+	private String ultimoCepilladoToString(Mascota mascota) {
+
+		String capillado = mascota.getHigiene().getCepillado();
+
+		return capillado;
+
+	}
+
+	private String desparasitadoToString(Mascota mascota) {
+
+		String returnDesparasitado = "";
+
+		Map<String, String> desparasitadoMascota = mascota.getHigiene().getDesparasitado();
+
+		for (Map.Entry<String, String> v : desparasitadoMascota.entrySet()) {
+
+			returnDesparasitado += "Tipo: " + v.getKey() + "\n" + "Fecha Desparasitación: " + v.getValue() + "\n\n";
+
+		}
+
+		return returnDesparasitado;
+
+	}
+
+	private String productosToString(Mascota mascota) {
+
+		String returnProductos = "";
+
+		ArrayList<String> productosMascota = mascota.getHigiene().getProductosNR();
+
+		for (String producto : productosMascota) {
+
+			returnProductos += "Producto: " + producto + "\n";
+
+		}
+
+		return returnProductos;
+
+	}
+
+	public void initialize() throws MalformedURLException {
+
+		if (mascotas.isEmpty()) {
+			// Nothing to do here.
+			return;
+		}
+
+		if (mascotas.size() >= 1) {
+
+			setProfile(petImage0, petName0, mascotas.get(0));
+
+			ultimoBano.setText(ultimoBanoToString(mascotas.get(0)));
+
+			ultimoCepillado.setText(ultimoCepilladoToString(mascotas.get(0)));
+
+			ultimaDesp.setText(desparasitadoToString(mascotas.get(0)));
+
+			productosNR.setText(productosToString(mascotas.get(0)));
```

Comment:
Estas 4 líneas son equivalentes a `selectProfile(0)`, no?
```suggestion
			selectProfile(0);
```